pages/en/lb4/Concepts.md

```diff
@@ -15,7 +15,7 @@ LoopBack 4 introduces some new concepts that are important to understand:
 - [**Schema**](Schemas.html): Represents the definition of a model in LoopBack, with respect to the datasource juggler. The `@loopback/repository` module provides special decorators for adding metadata to TypeScript/JavaScript classes to use them with the legacy implementation of Datasource Juggler.
 - [**Controller**](Controllers.html): Class that implements operations defined by application’s REST API. It implements an application’s business logic and acts as a bridge between the HTTP/REST API and domain/database models. A Controller operates only on processed input and abstractions of backend services / databases.
 - [**Context**](Context.html): An abstraction of all state and dependencies in your application, that LoopBack uses to “manage” everything. It's a global registry for everything in your app (configurations, state, dependencies, classes, and so on).
-- [**Dependency Injection**](Dependency Injection.html): Technique that separates the construction of dependencies of a class or function from its behavior, to keep the code loosely coupled.
+- [**Dependency Injection**](Dependency-injection.html): Technique that separates the construction of dependencies of a class or function from its behavior, to keep the code loosely coupled.
```

Comment:
Should be `Dependency-Injection.html` (capital I), once it's fixed, LGTM

Reply:
The URL is http://loopback.io/doc/en/lb4/Dependency-injection.html so the link should be `[**Dependency Injection**](Dependency-injection.html)`.  This may have changed recently.